packages/e2e-tests/specs/plugins/inner-blocks-allowed-blocks.test.js

```diff
@@ -76,6 +77,7 @@ describe( 'Allowed Blocks Setting on InnerBlocks ', () => {
 		) )[ 0 ];
 		await insertButton.click();
 		await insertBlock( 'Image' );
+		await page.click( listSelector );
```

Comment:
Could you detail this change?

Reply:
As far as I can remember, the focus was not sent to the correct place after the image was inserted which caused the test to fail. I can't remember anything else.

Reply:
Reverted and fixed, see [here](https://github.com/WordPress/gutenberg/pull/16708#issuecomment-543568335)